versions.tf

```diff
@@ -0,0 +1,13 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "4.50.0"
+    }
+    github = {
+      source  = "integrations/github"
+      version = "~> 5.12"
+    }
+  }
+  required_version = ">= 1.3.2"
```

Comment:
Should upgrade this to the latest Terraform version. You will need to install that version on your local machine as well. I put some notes about running different versions of Terraform on the same machine on Slack.

Reply:
Upgraded it, I'll add the instructions on the readme file in the next PR